app/src/main/res/layout/fragment_splash.xml

```diff
@@ -19,7 +19,8 @@
             app:layout_constraintEnd_toEndOf="parent"
             app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintTop_toTopOf="parent"
-            app:srcCompat="@drawable/ic_california_bear" />
+            app:srcCompat="@drawable/ic_california_bear"
+            android:contentDescription="@string/splash_screen_header_content_description" />
```

Comment:
I don't think this is something we need.
This is not actionable view: meaning we don't click and interact with it. 
In the use case when a blind person navigates through the app, a word "California" doesn't explain much.

Reply:
It should be `State of California`, but it is important visual information on the screen. It's a branding element that conveys the official status of the app unlike the other graphics on the screen.